css/style.css

```diff
@@ -72,4 +72,100 @@ li{
     -ms-border-radius: 50%;
     -o-border-radius: 50%;
 }
-/***** End general *****/
\ No newline at end of file
+/***** End general *****/
+/***** hero *****/
+.hero{
+    height: 100vh;
+    background: linear-gradient(
+        rgba(0,0,0,.3), 
+        rgba(0,0,0,.3)),url(../assets/images/hero-img.jpeg) no-repeat 50%;
+    background-size: cover;
+    position: relative;
+    z-index: 1;
+}
+.hero .flexContainer{
+    flex-direction: column;
+    height: 100%;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+}
+.heroHeader{
```

Comment:
font is not applied to the header

Reply:
yes I changed the "font-family" line of the body on the dev branch I just need to pull the changes to be applied

Reply:
then merge dev into your current branch, should fix it